gradle.properties

```diff
@@ -0,0 +1,26 @@
+#------------------------------------------------------------------------------
+# The empty properties in this file indicate you should define and set them
+# in ${HOME}/.gradle/gradle.properties.  The empty properties are required
+# when running specific tasks.
+#
+# These properties are overridable from the command line using the -P or -D, or
+# using an environment variable in the form ORG_GRADLE_PROJECT_<property-name>.
+#------------------------------------------------------------------------------
+description="Common code shared across IMRT projects."
+group=org.opentestsystem.ap
+
+version=0.0.1-SNAPSHOT
```

Comment:
Should this be `0.1.0-SNAPSHOT`?

Reply:
0.1.0 would be consistent with IIS and ISS

Reply:
Yeah, I noticed it after I pushed this PR and wanted to double check.  I'll make the change.  Thanks.